lib/Runtime/Language/JavascriptOperators.cpp

```diff
@@ -641,6 +641,13 @@ using namespace Js;
         TypeId leftType = JavascriptOperators::GetTypeId(aLeft);
         TypeId rightType = JavascriptOperators::GetTypeId(aRight);
 
+        if ((leftType == TypeIds_BigInt) || (rightType == TypeIds_BigInt))
+        {
+            // TODO: support comparison with types other than BigInt
+            AssertMsg(leftType == rightType, "do not support comparison with types other than BigInt");
```

Comment:
AssertOrFailFastMsg is preferable for things that aren't implemented because it will crash in release mode too. Much safer than passing arbitrary data to a function which will cast it to BigInt and read the data.